### ISSUE TITLE
Errors creating Pods

### ISSUE THREAD

**Issue by CelsoSantos** [GoogleCloudPlatform/continuous-deployment-on-kubernetes#25]
Hi,

I've been following both the Google Guide (from GCE) and this one and my pod never starts. It just throws out an error on the deployment stating this:
`Unable to mount volumes for pod "jenkins-2539707196-u95xz_jenkins(2bed9074-693e-11e6-9c2d-42010af00181)": timeout expired waiting for volumes to attach/mount for pod "jenkins-2539707196-u95xz"/"jenkins". list of unattached/unmounted volumes=[jenkins-home]
Error syncing pod, skipping: timeout expired waiting for volumes to attach/mount for pod "jenkins-2539707196-u95xz"/"jenkins". list of unattached/unmounted volumes=[jenkins-home]`

I've been stuck with this error for quite some time and I'm absolutely positive that the disk is there and is not attached to any VM upon execution of the deployment file.

I've even tried attaching the disk to a VM and formatting it first (then unmount and dettach) but I keep hitting the same error.

Is there any corrections that need to be done on the deployment file?

**Comment by viglesiasce** [comment]
I haven't been able to reproduce this issue. Is this with a new or existing cluster? Is your cluster multizone? If so the pod could be landing in a zone that does not contain the volume.

**Comment by CelsoSantos** [comment]
Hello Vic,

I'll have to check but I'll definitely get back to you on this.
I think the cluster is europe-west1-d only.

Later tonight I'll do some more tests and see this leads.

**Comment by viglesiasce** [comment]
Sounds good. Keep me posted.

**Comment by CelsoSantos** [comment]
Hello @viglesiasce ,

I'm sorry for not replying earlier, but I really just didn't have the time to give this another try.

I just did now, and I did everything "by the book" following each and every one of the commands on the README and I can report that they work and I now have a Jenkins deployment running.
I had previously been using my locally installed gcloud and that may have caused issues, but at least running from the Cloud Shell, everything is as is was supposed.

I can't vouch for running the included application since I just really wanted to have jenkins working and I now have so, I've stopped the rest of the process. 

I'm still going to try do delete everything and re-do the process, this time using europe-west and I'll report back on the success of that, but for now, it seems to be A-OK :)

Best Regards
